### ISSUE TITLE
Overwritten configs.

### ISSUE THREAD

**Issue by Zoubiey** [lenosisnickerboa/csgosl#33]
Possible to add an option if you want the GUIs option to overwrite the "hard" edited files or not? 
I'll rather just use the GUI to start the server with some basic stuff like max players, lan only and do all the settings by myself in the files.

Some of the settings doesn't even seem to work. Like roundtime is set to 3min in both GUI and server.cfg and it still is 5min rounds. Bots are set to 0 in both GUI and server.cfg and still gives me 5-6 bots in each team on connect.

Also maybe an option to use the admin_simple.cfg instead of GUI, seems much tougher to add admins with specific flags and immunity in the GUI but that might just be me.

**Comment by lenosisnickerboa** [comment]
I've been thinking about how to better coexist with the settings in the original cfg files and the idea is to leave all settings untouched which csgosl doesn't control. In case csgosl controls a setting it's harder to avoid overwriting original cfg files. I've been thinking about some kind of tick box to enable/disable certain settings in the gui, e.g. if you would untick the "round time" setting this setting would remain untouched in the original config files. Not sure how messy stuff would be if I introduced this for all options. The main reason for csgosl is simplicity and the target audience is people who don't want to mess with the config files themselves. I need to think about this some more...

About round time I haven't seen this bug, will check if I can reproduce it. Will open a certain issue for this.

About bot count, is it possible you have number players set to 6 and the "fillwithbots" option enabled. In that case you will not get zero bots even if botcount is set to zero since fillwithboths will add bots to the stated number of players.

**Comment by Zoubiey** [comment]
Kinda new to linux hosting and still havent managed to get a grip around all the cfg files. They are exactly the same as windows ofc but I havent needed to ever touch gamemode_ file and seems like the overwrote some settings (maybe the roundtime, havent checked in on that).

The bot count I tried with 0 and unticked I also have set the bot settings inside server.cfg. Gonna test around a bit more without csgosl and see if I managed atleast get the server up and running correctly before I swtich to csgosl (feel good with a GUI, I just love SteamCMD GUI for Windows)!
